src/scss/skin-modern/components/_label.scss

```diff
@@ -6,6 +6,7 @@
   @include hidden;
 
   cursor: default;
+  padding: .25em;
```

Comment:
Do you think it would be possible to scale the padding via a global skin variable (in `_variables.scss`)? E.g. `$padding: .25em !default;` or `$padding-scale: 1 !default;`.

Reply:
Hm, it would increase complexity to positioning the `seekbar` / `volumeslider`.